guide/install-manual.md

```diff
@@ -0,0 +1,202 @@
+# Installing Moonfire NVR
+
+This document describes how to install Moonfire NVR on a Linux system.
+
+## Downloading
+
+See the [github page](https://github.com/scottlamb/moonfire-nvr) (in case
+you're not reading this text there already). You can download the bleeding
+edge version from the command line via git:
+
+    $ git clone https://github.com/scottlamb/moonfire-nvr.git
+
+## Building from source
+
+There are no binary packages of Moonfire NVR available yet, so it must be built
+from source.
+
+Moonfire NVR is written in the [Rust Programming
+Language](https://www.rust-lang.org/en-US/). In the long term, I expect this
+will result in a more secure, full-featured, easy-to-install software.
+
+You will need the following C libraries installed:
+
+* [ffmpeg](http://ffmpeg.org/) version 2.x or 3.x, including `libavutil`,
+  `libavcodec` (to inspect H.264 frames), and `libavformat` (to connect to RTSP
+  servers and write `.mp4` files).
+
+  Note ffmpeg library versions older than 55.1.101, along with all versions of
+  the competing project [libav](http://libav.org), don't not support socket
```

Comment:
I think this is my typo, but as long as you're touching this, could you s/don't not support/don't support/?